api/database/helpers/DirectiveFormsApiEnums.php

```diff
@@ -0,0 +1,367 @@
+            self::OPERATIONS_CONSIDERATION_FUNDING_SECURED_COST_RECOVERY_BASIS,
+            self::OPERATIONS_CONSIDERATION_UNABLE_CREATE_NEW_INDETERMINATE,
+            self::OPERATIONS_CONSIDERATION_UNABLE_CREATE_NEW_TERM,
+            self::OPERATIONS_CONSIDERATION_UNABLE_CREATE_CLASSIFICATION_RESTRICTION,
+            self::OPERATIONS_CONSIDERATION_STAFFING_FREEZE,
+            self::OPERATIONS_CONSIDERATION_OTHER,
+        ];
+    }
+
+    const CONTRACTING_RATIONALE_SHORTAGE_OF_TALENT = 'SHORTAGE_OF_TALENT';
+    const CONTRACTING_RATIONALE_TIMING_REQUIREMENTS = 'TIMING_REQUIREMENTS';
+    const CONTRACTING_RATIONALE_HR_SITUATION = 'HR_SITUATION';
+    const CONTRACTING_RATIONALE_FINANCIAL_SITUATION = 'FINANCIAL_SITUATION';
+    const CONTRACTING_RATIONALE_REQUIRES_INDEPENDENT = 'REQUIRES_INDEPENDENT';
+    const CONTRACTING_RATIONALE_INTELLECTUAL_PROPERTY_FACTORS = 'INTELLECTUAL_PROPERTY_FACTORS';
+    const CONTRACTING_RATIONALE_OTHER = 'OTHER';
+
+    /**
+     * A collection of enums for ContractingRationale in factories and seeders
+     *
+     * @return string[]
+     */
+    public static function contractingRationales(): array
+    {
+        return [
+            self::CONTRACTING_RATIONALE_SHORTAGE_OF_TALENT,
+            self::CONTRACTING_RATIONALE_TIMING_REQUIREMENTS,
+            self::CONTRACTING_RATIONALE_HR_SITUATION,
+            self::CONTRACTING_RATIONALE_FINANCIAL_SITUATION,
+            self::CONTRACTING_RATIONALE_REQUIRES_INDEPENDENT,
+            self::CONTRACTING_RATIONALE_INTELLECTUAL_PROPERTY_FACTORS,
+            self::CONTRACTING_RATIONALE_OTHER,
+        ];
+    }
+
+    const PERSONNEL_TELEWORK_OPTION_FULL_TIME = 'FULL_TIME';
+    const PERSONNEL_TELEWORK_OPTION_PART_TIME = 'PART_TIME';
+    const PERSONNEL_TELEWORK_OPTION_NO = 'NO';
+
+    /**
+     * A collection of enums for PersonnelTeleworkOption in factories and seeders
+     *
+     * @return string[]
+     */
+    public static function personnelTeleworkOptions(): array
+    {
+        return [
+            self::PERSONNEL_TELEWORK_OPTION_FULL_TIME,
+            self::PERSONNEL_TELEWORK_OPTION_PART_TIME,
+            self::PERSONNEL_TELEWORK_OPTION_NO,
+        ];
+    }
+
+    const PERSONNEL_SKILL_EXPERTISE_LEVEL_BEGINNER = 'BEGINNER';
+    const PERSONNEL_SKILL_EXPERTISE_LEVEL_INTERMEDIATE = 'INTERMEDIATE';
+    const PERSONNEL_SKILL_EXPERTISE_LEVEL_EXPERT = 'EXPERT';
+    const PERSONNEL_SKILL_EXPERTISE_LEVEL_LEAD = 'LEAD';
+
+    /**
+     * A collection of enums for PersonnelSkillExpertiseLevel in factories and seeders
+     *
+     * @return string[]
+     */
+    public static function personnelSkillExpertiseLevels(): array
+    {
+        return [
+            self::PERSONNEL_SKILL_EXPERTISE_LEVEL_BEGINNER,
+            self::PERSONNEL_SKILL_EXPERTISE_LEVEL_INTERMEDIATE,
+            self::PERSONNEL_SKILL_EXPERTISE_LEVEL_EXPERT,
+            self::PERSONNEL_SKILL_EXPERTISE_LEVEL_LEAD,
+        ];
+    }
```

Comment:
I believe this enum will be useful when we start the skills library. Might be one we can move to a more generic namespace. Also, wouldn't hurt to drop the `PERSONNEL_` prefix too.

Reply:
Yeah, I was definitely thinking about that here.  I wasn't confident that the enums would exactly match up which is why I namespaced them with the prefix.  Other ones like security screening and work language are very different from the ones we use in the main app.

Reply:
Makes sense, if they turn out to be identical we can always move it in the future 👍